### ISSUE TITLE
items that are buoyant move with water currents when thrown into a river or the ocean or a lake or something

### ISSUE THREAD

**Issue by TiberianEuan** [CleverRaven/Cataclysm-DDA#62334]
### Is your feature request related to a problem? Please describe.

I wish I could throw a bottle with paper in it into the ocean for some random zombie to trip over when it reaches the shore!

### Solution you would like.

Instead of sitting there doing nothing in the water, it should float away!

### Additional context

https://youtu.be/mG_M-sj2CNA?t=121

**Comment by ZhilkinSerg** [comment]
There is no ocean and/or currents.

**Comment by Zireael07** [comment]
Rivers have currents IRL, I think that's what OP is referring to.

**Comment by NetSysFire** [comment]
This is a feature request for a feature that does not exist yet though

**Comment by TiberianErin** [comment]
rivers and lakes exist in cdda

**Comment by NetSysFire** [comment]
Yes, they do exist but there is no concept of direction for flowing water at all. In order for this to be implemented, currents need to exist. This will be an exceptionally complex task, potentially involving water physics which also do not exist yet.

Deep water will also simply swallow items and moving items around in a random direction in shallow (flowing) water, which would be possible right now, is kind-of counter productive.

Keep in mind that this is a cool idea but someone needs to do the work. And it will be a giant load of it, possibly involving major changes to how worlds work at the moment.

**Comment by TiberianErin** [comment]
what about while there is no water physics, and objects just float in the direction they were thrown or randomly if dropped.

**Comment by oosyrag** [comment]
> what about while there is no water physics, and objects just float in the direction they were thrown or randomly if dropped.

That would be rudimentary water physics, albeit unrealistic, unintuitive, and potentially really annoying (imo) from a gameplay perspective.
